src/main/telemetry/jetiexbus.c

```diff
@@ -299,32 +387,37 @@ void handleJetiExBusTelemetry(void)
     }
 }
 
-void sendJetiExBusTelemetry(uint8_t packetID)
+
+
+uint8_t sendJetiExBusTelemetry(uint8_t packetID, uint8_t item)
 {
-    static uint8_t sensorDescriptionCounter = 0;
-    static uint8_t sensorValueCounter = 1;
+    static uint8_t sensorDescriptionCounter = 0xFF;
     static uint8_t requestLoop = 0;
     uint8_t *jetiExTelemetryFrame = &jetiExBusTelemetryFrame[EXBUS_HEADER_DATA];
-
-    if (requestLoop == 100) {              //every nth request send the name of a value
-        if (sensorDescriptionCounter == JETI_EX_SENSOR_COUNT )
-            sensorDescriptionCounter = 0;
-
+    
+    if (requestLoop < 0xFF) {
```

Comment:
Is `requestLoop` reset somewhere?

Reply:
No, it's just for a run on 0xFF

Reply:
Old code did send ExTelemetryTextMessage on each hundredth call. New code sends  ExTelemetryTextMessage for first 255 invocations, then start sending values only (createExTelemetryValueMessage). 

What if transmitter is restarted?

Reply:
The parameter names are stored in the transmitter. The names are only necessary once. If a new sensor is available, it will be made available in the transmitter.
The variant of cyclic transfer is actually not necessary. The sensors do not change that often, so the names are only transmitted at the beginning.